### DISCUSSION TITLE
Flyway migration for multiple database

### DISCUSSION THREAD

**Discussion by akumar074** [quarkusio/quarkus#30592] [Q&A]
Does `quarkus-flyway` extension support the vendor-based script selection for migration like what we have in `spring-boot`?

example:
```
spring:
  flyway:
    locations: "classpath:db/migration/{vendor}"
```

[a comment by an automated account is not shown]

**Comment by geoand** [comment]
Not as far as I remember. Do you have an example of how this is used in Spring Boot?

**Comment by akumar074** [comment]
@geoand Thanks for the quick reply. It's mentioned here in spring [docs](https://docs.spring.io/spring-boot/docs/2.1.1.RELEASE/reference/html/howto-database-initialization.html#howto-execute-flyway-database-migrations-on-startup).

**Reply by geoand** [reply]
Interesting thanks. I was not aware of that feature.

We could certainly add it to Quarkus, feel free to open an issue asking for it

**Reply by akumar074** [reply]
Thanks. Here's the issue created https://github.com/quarkusio/quarkus/issues/30650.
